src/daft-core/src/python/series.rs

```diff
@@ -136,6 +136,10 @@ impl PySeries {
         Ok(self.series.floor_div(&other.series)?.into())
     }
 
+    pub fn pow(&self, exp: f64) -> PyResult<Self> {
+        Ok(self.series.pow(exp)?.into())
+    }
+
```

Comment:
series should delegate to the expressions instead of exposing it's own bindings. [reference](https://github.com/Eventual-Inc/Daft/blob/main/CONTRIBUTING.md?plain=1#L333)

see [Adding New Expressions](https://github.com/Eventual-Inc/Daft/blob/main/CONTRIBUTING.md#adding-new-expressions) in CONTRIBUTING.md for more information